### ISSUE TITLE
Patreon Benefits (Clothing/Vehicles)

### ISSUE THREAD

**Issue by gxdsent** [fkk-cz/noire-public#1038]
First off, I’m just going to speak for a lot of people who are afraid to speak up publicly, but it needs to be addressed.

For a few months now, we were just told requests for vehicles would be put on hold to fix some issues, and then next the clothing were completely pulled as well. Everyone has been left clueless on what is going on because of the lack of transparency and people are still paying for tiers of Patreon thinking they’re going to have a chance to request some new stuff for the city they pay for, but month after month no updates they’re blindly paying in hopes something changes, including myself personally. We all notice things being added and changed for certain people and for everyone else they’re shit outta luck.

If we’re being honest, it’s fucked up in the grand scheme of things. At least if you know you’re not going to attempt to fix the issues in a timely manner or try to find/hire people on to help you dissolve the issues then at least adjust the pricing to reflect what people are getting or add different categories that are just for like prio/housing/etc. AT THE LEAST, add in some sort of status or updates that will let people know what to expect on the backend of things in regards to issues like the custom clothing/vehicles.

Hopefully this can be addressed ASAP as many people have been upset that they’re not getting any insight onto why they don’t know what’s going on.

**Comment by TdotG** [comment]
Agree with you man. I haven’t been able to request any cars for some time now. It just seems pointless to buy Gold/Diamond if We’re not gonna be able to use the benefits. At this point we’re paying for queue.

**Comment by TheBottomFragger** [comment]
Something definitely needs to get added to the patreon page that says that it’s put on hold. People keep paying for for example diamond, thinking they can request cars and clothes, but when they ask about it, they get told that it is not getting accepted atm. Kinda sucks to get told that AFTER paying for it, especially for newer people

**Comment by gxdsent** [comment]
> Something definitely needs to get added to the patreon page that says that it’s put on hold. People keep paying for for example diamond, thinking they can request cars and clothes, but when they ask about it, they get told that it is not getting accepted atm. Kinda sucks to get told that AFTER paying for it, especially for newer people

exactly! when i hear about it, just makes me feel bad for em bc they don’t know it’s been like that for months

**Comment by HvrdNo** [comment]
tbh yall just paying for a garage that shouldnt be paywalled you already paid for the spot
if we are being honest the whole patron benefits should be reworked

**Comment by HvrdNo** [comment]
my bad forgot about the queue prio and ped skins and if we are being honest prio isnt needed from seeing the numbers recently

**Comment by AustinM0527** [comment]
> First off, I’m just going to speak for a lot of people who are afraid to speak up publicly, but it needs to be addressed.
> 
> For a few months now, we were just told requests for vehicles would be put on hold to fix some issues, and then next the clothing were completely pulled as well. Everyone has been left clueless on what is going on because of the lack of transparency and people are still paying for tiers of Patreon thinking they’re going to have a chance to request some new stuff for the city they pay for, but month after month no updates they’re blindly paying in hopes something changes, including myself personally. We all notice things being added and changed for certain people and for everyone else they’re shit outta luck.
> 
> If we’re being honest, it’s fucked up in the grand scheme of things. At least if you know you’re not going to attempt to fix the issues in a timely manner or try to find/hire people on to help you dissolve the issues then at least adjust the pricing to reflect what people are getting or add different categories that are just for like prio/housing/etc. AT THE LEAST, add in some sort of status or updates that will let people know what to expect on the backend of things in regards to issues like the custom clothing/vehicles.
> 
> Hopefully this can be addressed ASAP as many people have been upset that they’re not getting any insight onto why they don’t know what’s going on.

I couldn’t agree more with everything said here. If was a big part on why I canceled my gold/diamond. No point in having it when I can’t have the benefits of paying for it. Hope something changes asap!

**Comment by fkk-cz** [comment]
Both clothing and vehicle requests were placed on hold for separate reasons. For the former, people were being unreasonable and attaching, in some cases, dozens of items in one submission which slowed the entire process down. This was unfair to individuals who were much more measured and only wanted a few items added. Vehicle requests were closed due to the amount of game crashing caused by those compressed `script_rt` textures -- something that I've fixed by writing a little tool to help expedite the process.

I've reopened both now that there are more people (other than myself) who are able to look at and respond to incoming requests. Apologies for the lack of status updates or movement on this -- things that are placed on hold tend to stay there longer than they should due to complacency and other issues that take up a lot of time and attention. Ideally all requests should be either answered & closed within a timely manner, this is something we hope to improve going forward.
